scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -789,10 +789,11 @@ object Dialect extends InternalDialect {
   }
   private lazy val standardPairs = Seq[sourcecode.Text[Dialect]](
     Dotty,
+    Scala3x,
```

Comment:
Scala3Future ?

Reply:
sorry, pulled the trigger too early. let me know if you'd like me to rename; tried to make it shorter (plus, some repos -- such as intellij -- use `x` to denote a future version).